lib/charms/karma_k8s/v0/karma.py

```diff
@@ -242,50 +241,50 @@ class KarmaConsumer(ConsumerBase):
     metadata.yaml file:
 
     ```yaml
-    requires:
+    provides:
       karma-dashboard:
         interface: karma_dashboard
     ```
 
     A typical example of importing this library might be
 
     ```python
-    from charms.karma_k8s.v0.karma import KarmaConsumer
+    from charms.karma_k8s.v0.karma import KarmaProvider
     ```
 
     In your charm's `__init__` method:
 
     ```python
-    self.karma_lib = KarmaConsumer(
+    self.provider = KarmaProvider(
         self,
         "karma-dashboard",
-        consumes={"karma": ">=0.86"},
+        ">=0.86",
     )
     ```
```

Comment:
I don't think this is right. Provider should except (name, service, version). And version should be exact.

Reply:
Indeed.

Reply:
Nice spotting! Fixed.